docs/api/js/utils.md

```diff
@@ -13,7 +13,12 @@ Most of the utilities are used internally by the zkSync team. So this document w
 
 ## The "address" of ether
 
-While formally ether is a token with address `0x000000000000000000000000000000000000800a` on zkSync, we use the "zero address" as a more user-friendly alias:
+While *de facto* ether is a token with address
+
+```typescript
+export const L2_ETH_TOKEN_ADDRESS = "0x000000000000000000000000000000000000800a";
+```
+ on zkSync, *de jure* in SDK and API "zero address" is used as a more user-friendly alias:
```

Comment:
we are not a country, there is no *jure* here : )

Reply:
should I say ` formally`?

Reply:
I would rather put it like, "while Ether is actually a token deployed on the address `0x000000000000000000000000000000000000800a`, we also allow a more friendly zero address alias in our SDK and API"